test/snap/test-ejs-with-snapshot.js

```diff
@@ -2,7 +2,6 @@
 
 const t = require('tap')
```

Comment:
I see that tap is still being used here. Was this intentional? If so, it would have been useful if you had mentioned it in the PR description